src/anbox/cmds/session_manager.cpp

```diff
@@ -221,10 +227,13 @@ anbox::cmds::SessionManager::SessionManager()
 
     // The qemu pipe is used as a very fast communication channel between guest
     // and host for things like the GLES emulation/translation, the RIL or ADB.
+    const auto injector = di::make_injector(
```

Comment:
No, it is not required. It is another proposal to use in anbox, totally independent of sdbus-cpp. The purpose is to limit code that is not necessarily very productive (passing arguments and instantiating objects) while still keeping the code explicit (object has to list its dependencies). The library should be used in a number of places in anbox besides those that I used it in.

Are you against this approach in general or is it more about cost to benefit ratio?

Reply:
I am not against it in general but it adds another out-of-tree dependency we have to care and I don't think the small benefit outweighs that.

Reply:
Reverted for now. Maybe one day it will make sense to make another commit that reworks significant number of such cases so that it justifies the dependency on Boost::DI.